tool/flutter_ci_script_master.sh

```diff
@@ -19,6 +19,7 @@ declare -ar PROJECT_NAMES=(
     "code_sharing/server"
     "desktop_photo_search/fluent_ui"
     "desktop_photo_search/material"
+    "experimental/context_menus"
```

Comment:
When are we expecting this to be usable from `beta` channel? I'd like to add it as a commented out entry to the beta and stable CI scripts with TODO comments as to when it should be enabled.

Reply:
Will do.  It was merged 10 days ago (https://github.com/flutter/flutter/commit/0b451b6dfd6de73ff89d89081c33d0f971db1872).  I think it will make it into the next beta release?

Reply:
sgtm!